desktop/apps/consign/client/actions.js

```diff
@@ -45,6 +50,15 @@ export const UPDATE_SUBMISSION = 'UPDATE_SUBMISSION'
 export const UPDATE_USER = 'UPDATE_USER'
 
 // Action creators
+export function addAssetId (assetId) {
```

Comment:
We want to track the list of assets that are uploaded with a given submission. This lets us better understand when images are uploaded after-the-fact vs. during the submission flow.